### PR TITLE
Improve accessibility and other lighthouse scores for website

### PR DESCRIPTION
**Pull request by jMuzsik**
Hi,

I was just checking out the library after listening to you on the React Podcast the other day. Was awesome to see a very human perspective of the React team and open source in general. 

So, I went to the site and saw a slight problems on Google Lighthouse that I fixed up. Below is some pics of the difference in the lighthouse score:

Mobile test in my environment:

Screenshot 2019-07-18 at 12 57 26 PM
)

Mobile test for current site: 

Screenshot 2019-07-18 at 12 59 15 PM

And it's basically the same with desktop.

What I did:

1. Meta description
2. Slight changes in code styling and nav link styling

### CODE REVIEW

File: website/src/components/SubMenu.module.css [hunk 1]
```diff
@@ -29,7 +29,7 @@
 .SideNavLinkActiveDark {
   display: block;
   position: relative;
-  color: #ec5f67;
+  color: #9E0015;
```
[the remark this answers is not in the document]
Comment (jMuzsik) [reply website/src/components/SubMenu.module.css]: Oddly, this only pops up when you check Desktop for Device. So, reverting it won't fix the lighthouse problem. I used this to find it check what the proper color should be: 

https://webaim.org/resources/contrastchecker/

And something close to this color: #D21922 seems to be the lowest for 4.5 x 1 contrast ratio

So...

File: website/src/components/SubMenu.module.css [hunk 2]
```diff
@@ -13,11 +13,11 @@
 }
 
 .SideNavLinkListItem {
-  line-height: 2rem;
   padding-left: 1rem;
 }
 
 .SideNavLink {
+  line-height: 3.25rem;
```
[the remark this answers is not in the document]
Comment (jMuzsik) [reply website/src/components/SubMenu.module.css]: And this, I thought was extreme as well. It oddly, again, only pops up when you check Desktop even though the problem is a mobile problem. So, it's probably a slight error with the audit. It's on SEO in the Mobile Friendly section. I'll just revert this. Let me know what you think of the color thing, and I'll squash and push again soon

[the remark this answers is not in the document]
Comment (bvaughn) [reply website/src/components/SubMenu.module.css]: I'd rather have a worse lighthouse score than change this, TBH. So let's revert it. 😄

File: website/src/components/SubMenu.module.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (bvaughn) [reply website/src/components/SubMenu.module.css]: I would be okay with #D21922

**Comment by jMuzsik** [comment]
@bvaughn i implemented the changes!

**Comment by bvaughn** [comment]
Thanks

**Pull request merged**